components/claim/claim-card.tsx

```diff
@@ -8,7 +8,7 @@ import { useEffect } from "react";
 
 export interface ClaimCardProps {
   amount: string;
-  claimStatus: ClaimStatus;
+  claimStatus: string;
```

Comment:
is there a reason you can't use whatever is in `ClaimStatus`?

Reply:
I can probably update claimStatus to include the other values

Reply:
item.reason is a string value
but I am changing it